CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# specification-template
+## Bidra
+For å bidra til utviklingen av denne standarden, må du clone den til din datamaskin og opprette en pull request.
+
+### Clone til din datamaskin
+For at det følgende skal virke på din maskin, må du ha programvare installert:  
+ - [Git](https://git-scm.com/)
+ - en tekst-editor, feks [Atom editor](https://atom.io/)
+
+
+ Dersom du bruker Atom kan du gjøre som følger:  
+  - Trykk `Ctrl+Shift+P`
+  - Skriv `GitHub: Clone` og trykk Enter
+  - I Clone from legger du inn følgende url `https://github.com/Informasjonsforvaltning/specification-template.git`
+
+Alternativt i kommando-linje:
+```
+% git clone https://github.com/Informasjonsforvaltning/specification-template.git
+% cd specification-template
+% atom .                                # dersom du har installert Atom
+```
+### Gjøre endringer
+Denne standarden er laget i [AsciiDoc](http://asciidoc.org/) med hjelp av verktøyet [Asciidoctor](https://asciidoctor.org/).
+
+Alle endringer må gjøres ved å endre eller legge til filer i docs-folderen. Se for eksempel på følgende veileder for god AsciiDoc praksis: https://asciidoctor.org/docs/asciidoc-recommended-practices
+
+En typisk git-arbeidsflyt vil vere som følger:
+```
+% git checkout master
+% git pull
+% git checkout -b <ny branch>
+% ... # gjør endringer i din valgte editor
+% git add <filer som er endra>          # legger endringer klare for commit
+% git commit -m "Fornuftig feilmeldng"  # utføre commit
+% git push                              # publisere branch og gjøre klar for PR
+```
+Dersom du bruker Atom har du støtte for alt dette i Git-panelet.
+
+Gå til https://github.com/Informasjonsforvaltning/specification-template/pulls og opprett `New pull request`
+
+#### Forhåndsvisning
+Dersom du bruker [Atom editor](https://atom.io/) kan du installere pakken [AsciiDoc Assistant Package](https://atom.io/packages/asciidoc-assistant).
+Du vil da få syntax-uthevelse og forhåndsvisning.
+
+#### Generer html og pdf lokalt
+
+For at det følgende skal virke på din maskin, må du ha programvare installert:  
+ - [docker](https://www.docker.com/products/docker-desktop)
+
+```
+% docker run -it -v $(pwd):/documents asciidoctor/docker-asciidoctor
+bash-5.0# asciidoctor -D docs -o index.html docs/main.adoc
+bash-5.0# asciidoctor-pdf -D docs -o document.pdf docs/main.adoc
+bash-5.0# exit
+```
+Åpne filen index.html i din nettleser.
+
+#### Teste html i en webserver på din maskin
```

Comment:
Jeg tester visningen med asciidoc-preview, men den funker kun for enkeltfilene en jobber i. https://atom.io/packages/asciidoc-preview

Reply:
Funkar hos meg. Står i main.adoc:
![image](https://user-images.githubusercontent.com/1702357/83020040-3665d200-a028-11ea-970b-889e71a33fdb.png)

Reply:
Hehe, ja, det var bare et innspill til om vi skulle gi et tips om at det går an å preview på den måten også. Jeg har ikke forsøkt den måten du beskriver. Men den er kanskje like enkel.

Reply:
åja! Sorry, leste ikke riktig. Jeg har hatt problemer med å få til fra main men det skal jeg sannelig prøve igjen neste gang jeg får en main-fil!